content/en/blog/2023/experimental-dual-stack/index.md

```diff
@@ -0,0 +1,209 @@
+---
+title: "Support for Dual Stack Kubernetes Clusters"
+description: "Experimental support for Dual Stack Kubernetes Clusters."
+publishdate: 2023-03-02
+attribution: "Steve Zhang (Intel), Alex Xu (Intel), Iris Ding (Intel), Jacob Delgado (F5), Ying-chun Cai (formerly F5)"
+keywords: [dual-stack]
+---
+
+Over the past year, both Intel and F5 have collaborated on an effort to bring support for
+[Kubernetes Dual-Stack networking](https://kubernetes.io/docs/concepts/services-networking/dual-stack/) to Istio.
+
+## Background
+
+The journey has taken us longer than anticipated and we continue to have work to do. The team initially started with a design based
+on a reference implementation from F5. The design led to an [RFC](https://docs.google.com/document/d/1oT6pmRhOw7AtsldU0-HbfA0zA26j9LYiBD_eepeErsQ/edit?usp=sharing) that caused us to re-examine our approach. Notably, there were concerns about memory and performance issues that the community wanted
+to be addressed before implementation.
+
+## Experimental Dual Stack Branch
```

Comment:
I would prefer to remove this section in the blog - i think it generated more confusion. Not sure if this dev details would be interesting to users given now you do want people to use 1.17 branch.

Reply:
Removed the experimental branch section

Reply:
Thanks!